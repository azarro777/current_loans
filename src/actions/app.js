import data from '../utils/loans.json';
import { loadData } from '../reducers/appReducer';

export const fetchData = () => {
	return dispatch => {
		try {
			dispatch(loadData(data.loans));
		} catch (error) {
			console.log(error);
		}
	}
};