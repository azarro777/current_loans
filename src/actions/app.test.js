import React from 'react';
import { app } from './app';
import { shallow } from 'enzyme';

describe('Testing <app/>', () => {
	it('app have rendered correctly', () => {
		const app = shallow(<app/>);
		expect(app).toMatchSnapshot();
	});
});