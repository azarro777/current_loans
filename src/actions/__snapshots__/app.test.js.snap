// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <app/> app have rendered correctly 1`] = `ShallowWrapper {}`;
